.github/workflows/ci.yml

```diff
@@ -25,9 +25,6 @@ jobs:
   run_win_system_tests:
     needs: build_desktop
     uses: ./.github/workflows/run_win_system_tests.yml
-  run_ubuntu_system_tests:
```

Comment:
Updating the description saying that you are updating to ubuntu-22.04 for building and removing running the system tests on ubuntu as updating the AWS machine is causing some issues and we have the windows system tests to rely on along with internal testing that runs the same tests on both windows and ubuntu.